Documentation/Concepts/event_store.md

```diff
@@ -2,90 +2,189 @@
+
+The event store is an append-only database. Any events saved to it **cannot be changed** or **deleted**. It acts as the record of all events that have happened in the system from the beginning of time.
+
+Each [Tenant]({{< ref "tenants" >}}) has their own event store database, which is configured in [`resources.json`]({{< ref "docs/reference/runtime/configuration#resourcesjson" >}}).
+
+## Structure of the Event Store
+
+{{< tabs name="collections" >}}
+{{% tab name="MongoDB" %}}
+This is the structure of the event store implemented in [MongoDB](https://www.mongodb.com/). It includes the following collections in the default [Scope]({{< ref "#scope" >}}):
+
+- `event-log`
+- `aggregates`
+- `stream-processor-states`
+- `stream-definitions`
+- `stream-<streamID>`
+- `public-stream-<streamID>`
+
+
+Following JSON structure examples have each property's [BSON type](https://docs.mongodb.com/manual/reference/bson-types/) as the value.
+
+### `event-log`
+
+The Event Log includes all the [Events]({{< ref "events" >}}) committed to the event store in chronological order. All [streams]({{< ref "streams" >}}) are created from the event log.
+
+[Aggregate]({{< ref "aggregates" >}}) events have `"wasAppliedByAggregate":  true` set and events coming over the [Event Horizon]({{< ref "event_horizon" >}}) have `"FromEventHorizon": true"` set.
+
+This is the structure of a committed event:
+```json
+{
+    // this it the events EventLogSequenceNumber,
+    // which identifies the event uniquely within the event log
+    "_id": "decimal",
+    "Content": "object",
+    // Aggregate metadata
+    "Aggregate": {
+        "wasAppliedByAggregate": "bool",
+        // AggregateRootId
+        "TypeId": "UUID",
+        // AggregateRoot Version
+        "TypeGeneration": "long",
+        "Version": "decimal"
+    },
+    // EventHorizon metadata
+    "EventHorizon": {
+        "FromEventHorizon": "bool",
+        "ExternalEventLogSequenceNumber": "decimal",
+        "Received": "date",
+        "Concent": "UUID"
+    },
+    // the committing microservices metadata
+    "ExecutionContext": {
+        // 
+        "Correlation": "UUID",
+        "Microservice": "UUID",
+        "Tenant": "UUID",
+        "Version": "object",
+        "Environment": "string",
+    },
+    // the events metadata
+    "Metadata": {
+        "Occurred": "date",
+        "EventSource": "UUID",
+        // EventTypeId and Generation
+        "TypeId": "UUID",
+        "TypeGeneration": "long",
+        "Public": "bool"
+    }
+}
+```
+
+### `aggregates`
+
+This collection keeps track of all [Aggregate Roots]({{< ref "aggregates#aggregates-in-dolittle" >}}) registered with the Runtime.
```

Comment:
It keeps track of all Aggregates. Remember that for an AggregateRoot (a type of aggregate really) there can be multiple aggregates (with the event source as an unique identifier)

Reply:
Fixed it to just aggregates. But also by definition an Aggregate Root is the only entry point to a particular aggregate, it makes no sense for an aggregate root to contain multiple aggregates within it, no?